manifests/o/ossia/score/3.1.4/ossia.score.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ossia.score
+PackageVersion: 3.1.4
+PackageLocale: en-US
+Publisher: ossia
+PackageURL: https://ossia.io/

```

Comment:
```suggestion
PackageUrl: https://ossia.io/
```

Reply:
Sorry, I had incorrect capitalization in the previous suggestion. I've tested this suggestion to be sure it's correct.